recipe/meta.yaml

```diff
@@ -34,26 +32,27 @@ requirements:
     - {{ compiler('cgo') }}
     - {{ compiler('cxx') }}
     - {{ compiler("cuda") }}  # [cuda_compiler_version not in (undefined, "None")]
-    # omniscidb 5.4 is not LLVM 11 ready
-    - clangdev <11
+    # omniscidb 5.5 is not LLVM 12 ready
```

Comment:
note: maybe if we have this constraint `< 12` constantly, we could change it (in the future) to a variable (maybe LLVM_UNSUPPORTED_VERSION or something similar) and we could just change the variable value when needed.

Reply:
I would hope that at some point we could remove the constraint on the LLVM version altogether. Meanwhile, maintaining it within the meta.yaml is straightforward, and adding the extra boilerplate is not worth it, IMHO.